cmd/gateway/matic/matic.go

```diff
@@ -0,0 +1,16 @@
+package matic
+
+import (
+	"github.com/marlinprotocol/ctl2/cmd/gateway/matic/bor"
+	"github.com/spf13/cobra"
+)
+
+var MaticCmd = &cobra.Command{
+	Use:   "matic",
+	Short: "run matic chain gateways",
```

Comment:
Inconsistent..
ref:
```
Available Commands:
  cosmos      Cosmos Gateway
  dot         Polkadot Gateway
  iris        Iris Gateway
  matic       run matic chain gateways
  near        Near Gateway
  ```

Reply:
Can do `gateways` instead of gateway or some better way to differentiate. However remove verb `run`